clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -289,6 +289,9 @@ public void testInterceptorConstructorClose() throws Exception {
             consumer.close();
             assertEquals(1, MockConsumerInterceptor.INIT_COUNT.get());
             assertEquals(1, MockConsumerInterceptor.CLOSE_COUNT.get());
+            // Cluster metadata will only be updated on calling poll.
+            Assert.assertNull(MockConsumerInterceptor.CLUSTER_META.get());
```

Comment:
No. I have those tests in the integration tests. Doing a non-null test here will need a lot of mocking at the n/w layer. Do you think it adds value if it is already being tested in the integration tests ?